packages/translations/src/examples.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import i from 'i18next'
```

Comment:
Can you import this as something more readable? E.g. `i18n`

Reply:
done